spec/models/service_template_transformation_plan_task_spec.rb

```diff
@@ -556,9 +562,12 @@
       context 'destination is openstack' do
         let(:dst_ems) { FactoryBot.create(:ems_openstack, :api_version => 'v3', :zone => FactoryBot.create(:zone)) }
         let(:dst_cloud_tenant) { FactoryBot.create(:cloud_tenant, :name => 'fake tenant', :ext_management_system => dst_ems) }
-        let(:dst_cloud_volume_type) { FactoryBot.create(:cloud_volume_type) }
-        let(:dst_cloud_network_1) { FactoryBot.create(:cloud_network) }
-        let(:dst_cloud_network_2) { FactoryBot.create(:cloud_network) }
+        let(:dst_host) { FactoryBot.build(:host_openstack_infra, :ems_cluster => dst_cloud_tenant) }
+        # let(:dst_cloud_volume_type) { FactoryBot.create(:cloud_volume_type) }
+        let(:disk) { FactoryBot.create(:disk) }
+        let(:dst_cloud_volume_type) { FactoryBot.create(:cloud_volume_openstack, :attachments => [disk], :cloud_tenant => dst_cloud_tenant) }
```

Comment:
Is this variable name accurate?  It looks like it used to be a very different class.

Reply:
can't variable name be any supported string?  As far the class name is concerned, `cloud_volume_openstack` is a sub-class of `cloud_volume`

Reply:
Yes, it can.  I'm bringing it up because the commented out line above with the same name returned an instance of `CloudVolumeType` and this now returns an instance of `CloudVolume` and they are backed by two different classes and tables.

Reply:
`dst_cloud_volume_type` being used as a variable in name only even if it's `kind_of` is `cloud_volume_openstack`.  Left the same variable in-place because it was being used in more than one places.  I can circle back to it when implementing lan_validation

Reply:
refactored the code in the lan validations stream

Reply:
I still think something like `dst_cloud_volume` would be less confusing since it will be a CloudVolume, not a CloudVolumeType.

Reply:
Done.